README.md

```diff
@@ -93,22 +84,11 @@ as an [API user](https://signon.publishing.service.gov.uk/api_users).
 To authorise with a bearer token, a request has to be made with the header:
 
 ```
+# See https://github.com/alphagov/gds-api-adapters#app-level-authentication
```

Comment:
Not your change, but this URL no longer has an "App level authentication" section 🤔

Reply:
Oops 🙄. I've updated with a link to the code (doc seems to have gone 😢).